lib/net/dhcp/record.rb

```diff
@@ -41,7 +54,7 @@ def create
 
     # Returns an array of record objects which are conflicting with our own
     def conflicts
-      conflicts = [proxy.record(network, mac), proxy.records_by_ip(network, ip)].flatten.compact.delete_if { |c| c == self || related_macs.include?(c.mac) }
+      conflicts = [proxy.record(network, mac), proxy.records_by_ip(network, ip)].flatten.compact.delete_if { |c| c.lease? || c == self || related_macs.include?(c.mac) }
```

Comment:
This is THE change actually, if it's a lease type of record, filter it out. It is the first statement in the chain, most likely to happen.